styles.css

```diff
@@ -0,0 +1,90 @@
+/* .heading {
+  background-color: #b7e9f7;
+} */
+body {
+  background: #b7e9f7;
+}
+
+.app-wrap {
+  display: flex;
+  flex-direction: column;
+  max-width: 375px;
+  height: 130vh;
+  margin: 5vh auto 5vh auto;
+  border-radius: 15px;
+  background: #fff;
+}
+
+.app-header h1 {
+  width: 90px;
+  height: 28px;
+  padding-left: 5%;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  color: #28352f;
+}
```

Comment:
- [ ] Great job adding the typography as provided in the Figma design. But in order to match the design better, you should consider adding the right font to your project. In this case, you should have `inter` as your standard `font-family`.